src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -117,7 +118,7 @@ IFACEMETHODIMP_(void) FancyZones::Run() noexcept
     m_window = CreateWindowExW(WS_EX_TOOLWINDOW, L"SuperFancyZones", L"", WS_POPUP, 0, 0, 0, 0, nullptr, nullptr, m_hinstance, this);
     if (!m_window) return;
 
-    RegisterHotKey(m_window, 1, MOD_WIN, VK_OEM_3);
+    RegisterHotKey(m_window, 1, m_settings->GetSettings().editorHotkey.get_modifiers() | MOD_NOREPEAT, m_settings->GetSettings().editorHotkey.get_code());

```

Comment:
MOD_NOREPEAT should probably be built into the infrastructure. I'd make it the default and possibly provide a way for callers to specify they don't want it. Most/all hotkeys though do not want the repeat so it would be ok to not provide a way to override the behavior until necessary.